std/range/package.d

```diff
@@ -2688,16 +2695,124 @@ pure @safe nothrow @nogc unittest
     struct NonForwardRange
     {
         enum empty = false;
-        int front() { return 42; }
+        @property int front() { return 42; }
```

Comment:
The `@property` is useless here (same for the ones following). What was the motivation for adding it to the existing test?

Reply:
Standardization. Just about all the other front() routines in the file are `@property`. It was a superfluous change. I can undo if desired.

Reply:
The `@property` attributes have been removed.